app/controllers/concerns/sortable.rb

```diff
@@ -0,0 +1,13 @@
+module Sortable
+  extend ActiveSupport::Concern
+  
+  included do
+    def sort_column(column_to_sort)
+      column_to_sort.column_names.include?(params[:sort]) ? params[:sort] : ''
```

Comment:
use it something like following:

params[:sort] if column_to_sort.column_names.include?(params[:sort])